app/lib/backend_api_logic/routing_policy.rb

```diff
@@ -51,24 +51,16 @@ def as_json
               operations: [
                 match: :path,
                 op: :matches,
-                value: path_to_regex
+                value: config_path.to_regex
               ]
             }
           }.merge(replace_path)
         end
 
-        def path_to_regex
-          if path.to_s.empty?
-            "/.*"
-          else
-            "/#{path}/.*|/#{path}/?"
-          end
-        end
-
         def replace_path
-          return {} if path.blank?
+          return {} if config_path.blank?
```

Comment:
`blank?` internally calls the `empty?` of config_path (if it is a ConfigPath and not nil).

https://github.com/rails/rails/blob/e9d6b85f3e834ceea2aeabe4cbaa96a7c73eb896/activesupport/lib/active_support/core_ext/object/blank.rb#L4-L18

Just saying this because I found like black magic how this was working. And I see that it is well tested in:
https://github.com/3scale/porta/blob/fedd08b63974d66bece271d608704a8a053674b4/test/unit/backend_api_logic/routing_policy_test.rb#L46

Good :+1: